lib/teleterm/gateway/config.go

```diff
@@ -58,6 +60,8 @@ type Config struct {
 	KeyPath string
 	// Insecure
 	Insecure bool
+	// ClusterName
```

Comment:
The comments on which this one was based one were left there due to oversight. I think we should just remove them, similar to comments that follow the format of "Foo is foo" because they don't bring any value.